doc/develop/flash_debug/host-tools.rst

```diff
@@ -324,12 +324,10 @@ Check if your SoC is listed in `OpenOCD Supported Devices`_.
 
 .. note:: On Linux, openocd is available though the `Zephyr SDK
    <https://github.com/zephyrproject-rtos/sdk-ng/releases>`_.
-   Windows users should use the following steps to install
-   openocd:
+   To install OpenOCD on Windows, run ``choco install openocd`` as an Administrator.
+   On macOS, run ``brew install openocd``.
 
-   - Download openocd for Windows from here: `OpenOCD Windows`_
```

Comment:
Chocolatey appears to follow OpenOCD release schedule, which is not ideal to say the least (last one was in 03/2023), whereas gnutoolchains provides more up-to-date builds (20240916). So I think this should be kept.

Reply:
Zephyr used to ship OpenOCD in the SDK on Windows (as it is the case on Linux), I don't know why it's no longer the case.

Reply:
> Zephyr used to ship OpenOCD in the SDK on Windows (as it is the case on Linux), I don't know why it's no longer the case.

Host tools were never part of the Windows Zephyr SDK distribution.

We eventually plan to provide the host tools, including OpenOCD, on Windows and macOS; but, that likely will not be anytime soon (mainly because it is not a high priority item).

Reply:
Thank you for reviewing.
I restored this line.

Reply:
@soburi still deleted.

Reply:
I think it should be kept as an alternative to Chocolatey for people who want the latest build.
Also, from my experience, Chocolatey is a pain to use on corporate PCs bcz of firewalls/proxy...